update referral link path to swap page (#4)
galacticcouncil/HydraDX-web: faq.markdown

- Users don’t have to use asset in pairs, rather one or any number of assets
- Financial infrastructure for developers and their applications (e.g. swap assets directly in  wallet, you can plug it to dex aggregators or liquidity aggregator yearn.finance)

## [Who is the HydraDX Team?](#who-is-the-hydradx-team)

> The Zee Prime team has co-founded the project, which is led by [Jakub](https://twitter.com/GregusJakub). Other [Jakub](https://twitter.com/JakPanik) has joined as a technical co-founder and a CTO. Besides them, there are 4 other senior developers working on the project full time, plus two front end devs and a designer (see our [github repo](https://github.com/galacticcouncil/hack.HydraDX-node)). Then there is our talented and always ready to help community manager [Pavol](https://twitter.com/KovarikPalo). We are working with Blockscience on advanced mathematical modelling. 

Reviewer: This needs to be updated and is not very acurate. We'll try to do some writeup on this.

Reviewer 2: you mean the team section? 
if you tell me where to find more precise information i will rewrite it.

Reviewer: remove for now.

Reviewer: We need to write something much better :)

## [Why Substrate?](#why-substrate)

Substrate is a modular blockchain framework used in many other blockchains (Kusama, Polkadot, Edgeware, etc.) maintained by the vast community of developers. Our high scalability, predictable fees, possibility of implementing front running resistance - and many of our virtues are possible by using Substrate technology.

## [What is HDX token?](#what-is-hdx-token)

HDX is a native token of HydraDX. It will be used for governance, staking and more. We will disclose details when our model is validated.

## [What can I use HDX for?](#what-can-i-use-hdx-for)

- To minimize the trading fee cost
- For staking (PoS)
- Holders of HDX will have lower over-collateralization ratio
- Swag

## [How can I earn HDX?](#how-can-i-earn-hdx)

- Provide liquidity and be rewarded by earning HDX.
- Buy on HydraDX mainnet.
- Stake HDX and earn even more HDX
- Support ecosystem by taking part in bounties and getting paid from treasury

## [Can i buy HDX now?](#can-i-buy-hdx-now)

The seed round and LBP are over and right now there is no possibility to buy HDX tokens.
The token will become tradable in the mainnet.

> See how to earn HDX above.

Reviewer: link to faq question

## [What is the vesting schedule for the team and investors?](#what-is-the-vesting-schedule-for-the-team-and-investors)

The current vesting schedule stands as following:

> Investorshave a monthly cliff vesting period for 18 months since the mainnet, with the first 1/18 unlocked at the mainnet launch.  

Reviewer: ```suggestion
Investors have a vesting period of 18 months, with gradual unlocking, starting at the mainnet launch.

```

> The team/founders have additional 6 months to wait after mainnet launch to start vesting for 18 months (24 months total, monthly cliffs).

Reviewer: ```suggestion
The team and founders are vested for 18 months, starting 6 months after the mainnet launch with gradual unlocking. (24 months total)

```

Tokens are often needed to perform specific tasks; every parachain needs to accumulate $DOT for maintaining their parachain slot, or $ETH for smart contract calls/actions. HydraDX integration allows these entities to be able to buy these assets automatically when needed, without them having to hold big reserves upfront, making the financial operations more efficient. Investors can use Hydra to optimize their portfolio with continuous buying, using dollar cost averaging or automatic rebalancing.

## [What do I get for providing tokens and assets for liquidity pools?](#what-do-i-get-for-providing-tokens-and-assets-for-liquidity-pools)

- HDX & fees

## [Is there any exchange listings planned and when will the token be tradale?](#is-there-any-exchange-listings-planned-and-when-will-the-token-be-tradale)

> We are not actively seeking listings, the ultimate goal is to have HDX priced and traded in and around our native liquidity protocol - Omnipool. However, we do not discount the possibility that HDX will be listed on other exchanges. 

Reviewer: ```suggestion
We are not actively seeking listings. The ultimate goal is to have HDX priced and traded in and around our native liquidity protocol - Omnipool. However, we do not discount the possibility that HDX will be listed on other exchanges. 

```

Nevertheless, it is unlikely that the exchanges will list new Substrate-based tokens before parachain auctions. The early entry through the Balancer LBP was for people who believe in the team, the idea and the tech. 

Please keep in mind that tokens will likely not be tradable for some time (probably until mainnet plus couple of weeks). The tokens, however, will be immediately stake-able, generating rewards during the „unlisted” phase and beyond.

## [How are the tokenomics set?](#how-are-the-tokenomics-set)

> The tokenomics is in an advanced stage of being formalized. Given that HDX has some unique properties the team along with our partners at Blockscience are making sure that every detail and edge-case scenario is covered.  

Reviewer: ```suggestion
The tokenomics is in an advanced stage of being formalized. Given that HDX has some unique properties, the team, along with our partners at Blockscience, are making sure that every detail and edge-case scenario is covered.

```

Note, that the whole project is very long-term oriented so one can expect that the parameters will be aligned with such ambitions. We expect the details to be fully formalized and revealed in the next couple of weeks. 

The basics that we already know:

- 15% Founders, Investors & Advisors
- 5% LBP (proceeds will be used as a bootstrap for native liquidity on HydraDX Omnipool)
- 5% Incentivized Stakenet (bounties, staking rewards, etc. during Stakenet)
- 75% liquidity provision and staking rewards

The circulating supply at mainnet is expected to be ~25% total (out of which 60% are being vested). 

## [How are xHDX swapped into HDX?](#how-are-xhdx-swapped-into-hdx)

> xHDX is an ERC20 representation of HDX. xHDX will be redeemable as HDX 1:1 on the HydraDX Stakenet.

Reviewer: ```suggestion
xHDX is an ERC20 representation of HDX. xHDX will be redeemable as HDX 1:1 on the HydraDX Stakenet. After the claim, xHDX will be your early adopter badge on Ethereum forever.

```

- 5% Incentivized Stakenet (bounties, staking rewards, etc. during Stakenet)
- 75% liquidity provision and staking rewards

The circulating supply at mainnet is expected to be ~25% total (out of which 60% are being vested). 

## [How are xHDX swapped into HDX?](#how-are-xhdx-swapped-into-hdx)

xHDX is an ERC20 representation of HDX. xHDX will be redeemable as HDX 1:1 on the HydraDX Stakenet. After the claim, xHDX will be your early adopter badge on Ethereum forever.

The [claiming process](https://claim.hydradx.io/) is now open, please read the [guide](https://docs.hydradx.io/claim/) to claim.

## [What is the timeframe for launching HydraDX?](#what-is-the-timeframe-for-launching-hydradx)

- Hack-net [https://hack.hydradx.io](https://hack.hydradx.io)
- Incentivised testnet - Q1 2021
- Test-net with our pool model - early Q2 2021
- Mainnet 2021

## [How much did the project raise in the seed round?](#wow-much-did-the-project-raise-in-the-seed-round)

The company tasked with deploying the HydraDX chain has raised $1.5 million. Proceeds are being used for ongoing development of HydraDX.

## [Analysis of the Balancer LBP](#analysis-of-the-balancer-lbp)

> The final price was $0.08059 per xHDX. A total of 22.9m DAI has been raised as the initial liquidity for the HydraDX chain (of which 430k+ in fees). 87% of xHDX from the pool have been sold and the remaining 13% of xHDX (0.65% of total supply) will be moved to the HydraDX treasury.

Reviewer: ```suggestion
The final price was $0.08059 per xHDX. A total of ~22.9 million DAI has been raised as the initial liquidity for the HydraDX chain (of which 430k+ in fees). 87% of xHDX from the pool have been sold and the remaining 13% of xHDX (0.65% of total supply) will be moved to the HydraDX treasury.

```
